packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarTextButtonIOS.ios.js

```diff
@@ -50,9 +60,20 @@ const styles = StyleSheet.create({
   },
 });
 
+const tintColors = {
+  [BUTTON_TYPES.default]: colorGray700,
+  [BUTTON_TYPES.primary]: colorBlue500,
+};
+
+const disabledTintColors = {
+  [BUTTON_TYPES.default]: colorGray300,
+  [BUTTON_TYPES.primary]: colorBlue300,
```

Comment:
Have we had design input on this? I feel like this should be gray300 as well

Reply:
No, just my assumption. I don't agree but I'll get a designer to decide.

Reply:
Just to confirm that Chris decided to use gray300, so I've done that.